<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sectiondialog</class>
 <widget class="QDialog" name="sectiondialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Secci√≥n</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(243, 243, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>618</width>
        <height>788</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>50</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <widget class="QLabel" name="label_title">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>10</y>
            <width>431</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Montserrat</family>
            <pointsize>16</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(119, 102, 198);</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>80</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <widget class="QLabel" name="label_semester">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>131</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Montserrat</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(70, 70, 122);</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_semester">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>20</y>
            <width>231</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(70, 70, 122);</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>80</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <widget class="QLineEdit" name="lineEdit_option">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>20</y>
            <width>231</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(70, 70, 122);</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_option">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>131</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Montserrat</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(70, 70, 122);</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>360</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <widget class="QListWidget" name="listWidget">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>10</y>
            <width>390</width>
            <height>340</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(70, 70, 122);</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_subjects">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>131</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Montserrat</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(70, 70, 122);</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>100</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <widget class="QPushButton" name="pushButton_cancel">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>50</y>
            <width>141</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(222, 221, 218);
color: rgb(70, 70, 122);</string>
          </property>
          <property name="text">
           <string>Cancelar</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_ok">
          <property name="geometry">
           <rect>
            <x>410</x>
            <y>50</y>
            <width>141</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(119, 102, 198);
color: rgb(243, 243, 255);</string>
          </property>
          <property name="text">
           <string>Ok</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_question">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>571</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Montserrat</family>
            <pointsize>16</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(119, 102, 198);</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
